usage-examples/editorial-commands.ly

```diff
@@ -0,0 +1,34 @@
+\version "2.19.42"
+
+\include "scholarly/package.ly"
+
+\setOption scholarly.editorial.addition #`(
```

Comment:
Is this approach your actual idea or a temporary solution?

I have several concerns with it:

1)
This forces the end user to define the editorial functions, otherwise LilyPond will fail.

2)
The syntax with the necessary quasi- and unquoting is daunting for many users.

3)
This more or less restricts the feature to the usage of "simple" functions that can be conveniently expressed within that alist.

---

What I would suggest is something like

* use the `apply` syntax like it is implemented here
* "apply" one out of a number of editorial functions (which are more or less abstract for now)
* These functions are defined in another submodule of `ScholarLY`. So they can be used independently from the annotation interface.
* These functions make use of a lookup interface like the one here (implemented with `\setOption`) but by default provide functions for all (conceivable) grob types.